news/2023/2023-07-25-flag-change-updates.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Country Flag Change Updates"
+date: 2023-07-26 13:00:00 +0000
+---
+
+Ever felt like you need a flag change? Look no further.
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/banner.jpg)
+
+Flag changes have been disabled from the osu! community and from osu! support since December 2021, and for the first time since then they have been reopened — with some minor adjustments.
+
+### Changes
+
+For those who have had the opportunity to request a flag change in the previous system, you may remember having to email <accounts@ppy.sh> or <support@ppy.sh> and then wait for the response, to then send another email of the information they asked for. Quite a journey for a flag change that sometimes didn't fully resolve.
+
+Well, that system has been revamped and puts *most* of the work on the user! Woohoo! No intermediary.
+
+The new system has the flag changes right in with the account settings, also conveniently underneath the username. If the flag you possess does not match the country you are in, you'll get prompted to change it, shown in the images below.
+
+If you have the ability to change country flags the settings will look like:
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/user-pop-up.jpg)
+
+and subsequently, if you do not have the option to change flags it will appear like so:
+
+![](/wiki/shared/news/2023-07-25-flag-change-updates/unavailable-user-pop-up.jpg)
+
+### Regulations
+
+Changes like these wouldn't come without some regulations, so before you get all excited and change your flag to somewhere you don't really live in, read below:
+
+- There are 6 months of previous data (meaning you can't make an account and immediately change your flag).
+- At least 4 of the 6 months have logins from only one country location, with no changes from country to country.
+- You are not actively registered for any ongoing tournament.
```

Comment:
Not sure what "actively" registered means, I think you can just be registered or not be registered?

```suggestion
- You must not be registered for any ongoing tournament.
```